pkgs/development/libraries/qt-6/modules/qtbase.nix

```diff
@@ -389,24 +398,22 @@ qtbaseDrv = stdenv.mkDerivation rec {
     ''-${if libmysqlclient != null then "plugin" else "no"}-sql-mysql''
     ''-${if postgresql != null then "plugin" else "no"}-sql-psql''
 
-    "-make libs"
-    "-make tools"
+    "-make" "libs"
+    "-make" "tools"
```

Comment:
Does that make a difference? It shouldnt.

Reply:
> Does that make a difference?

im afraid no (did not fix the bug)

> It shouldnt.

it should ^^

how would i pass `-o "a b"` to configure? `configureFlags = [ ''-o "a b"'' ]`...?

Reply:
> how would i pass -o "a b" to configure? configureFlags = [ ''-o "a b"'' ]...?

I think that would need to be escaped but I am not fully sure:

```nix
configureFlags = [ "-o \"a b\"" ]
```